goonj/conf/channel_settings.py

```diff
@@ -0,0 +1,28 @@
+from goonj.channels import constants
+from goonj.channels.email_channel import EmailChannel
+from goonj.channels.slack_channel import SlackChannel
+
+
+class ChannelSettings(object):
+
+    def __init__(self, channels):
+        """
+
+        :param channels: given map of 'type of channels','list of  channels
+                         metadata' initializes
+                         {'channel_type',[{'channel_name','channel_object'}]
+        """
+        if constants.CHANNEL_EMAIL in channels:
```

Comment:
Let's review f2f. Didn't understand why we need to hardcode a dependency to a channel